exonum-java-binding/core/rust/src/utils/jni_cache.rs

```diff
@@ -27,6 +27,10 @@ use jni::{
 };
 use parking_lot::{Once, ONCE_INIT};
 use std::os::raw::c_void;
+use std::panic::catch_unwind;
```

Comment:
I'd suggest making it nested like the rest of the imports.

Reply:
:ok_hand: , but *why* do we use this syntax? It seems to be harder in case of merges — instead of a single line per imports rustfmt may compress several imports into a single line, if a single import disappears and expands than multiple lines would be affected in the diff.